src/keystats/keystats.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import { KeyStatsContainerStateProps } from './keystatsContainer'
+import { StatsLabel, StatsValue } from './keystatsLabel'
+import { KeyStats } from './types'
+
+const numberFormat = (num: number) => (
+  Number.isInteger(num)
+    ? num.toLocaleString()
+    : num.toLocaleString(undefined, {
+        maximumFractionDigits: 2,
+        minimumFractionDigits: 2
+    })
+)
+
+const KeyStats: React.SFC<KeyStatsContainerStateProps> = ({ keystats: { keystats } }) => {
+  const StatsMap = (keystat: KeyStats) => ({
```

Comment:
Maybe Keystats should not be concerned with the mapping of the different key/labels and values. I would suggest to move this function outside of this function to make this easy to read?

Reply:
I can do this in the branch where I'm converting the API to typescript, so you can assign a todo here for me.